sdk/storage/inc/datalake/datalake_utilities.hpp

```diff
@@ -12,9 +13,10 @@
 namespace Azure { namespace Storage { namespace DataLake { namespace Details {
   UrlBuilder GetBlobUriFromDfsUri(const UrlBuilder& dfsUri);
 
```

Comment:
These two functions will finally be removed, right?

Reply:
`GetBlobUriFromDfsUri` will be removed only after full interop is implemented.

Reply:
I meant `DeserializeMetadata` and `SerializeMetadata`